docker/runtime/Dockerfile.pytorch.aarch64

```diff
@@ -5,5 +5,5 @@ ARG WHEEL_FILE=torch_blade*.whl
 
 ADD ./build/${WHEEL_FILE}  /install/python/
 
-RUN source /opt/venv_disc/bin/activate && \
+RUN . /opt/venv_disc/bin/activate && \
```

Comment:
> The most obvious use case for using the dot command is when you want to change your current context by setting new variables or changing some existing one

ref: https://www.shell-tips.com/bash/source-dot-command/#gsc.tab=0

It seems that using the `dot` command may cause some unexpected behavior, how about installing the wheel with build-in Python?

Reply:
This is just a workaround. We have to install a special ARM Tensorflow wheel manually before we install disc whell package since community arm tf wheel is not compatible with disc.  And the special Tensorflow wheel is  only installed in venv environment currently.

Reply:
Better to add some command for this workaround, please feel free to add it in another PR.

Reply:
open an issue to track this: #260